Dockerfile

```diff
@@ -1,54 +1,38 @@
-FROM debian:9
+FROM debian:12-slim AS builder
 
 ENV DEBIAN_FRONTEND=noninteractive \
     DEBCONF_NONINTERACTIVE_SEEN=true \
     LC_ALL=C.UTF-8 \
     LANG=C.UTF-8
 
-COPY . /app
+RUN apt-get update \
+ && apt-get install -y \
+      build-essential \
+      autoconf \
+      libtool \
+      git \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
 
-# get dependencies, build, and remove anything we don't need for running jq.
-# valgrind seems to have trouble with pthreads TLS so it's off.
+WORKDIR /app
+COPY . /app
+RUN autoreconf -i \
+ && ./configure \
+      --disable-dependency-tracking \
+      --disable-silent-rules \
```

Comment:
Got a curious about these options, seems `--disable-silent-rules` is the default? if so maybe can skip specify it? `--disable-dependency-tracking` seems to speed up one-time builds by not generate make dependency files, i suspect not much build speed up for jq?

Reply:
I agree. I'll drop both flags.